lib/internal/test_runner/harness.js

```diff
@@ -71,16 +103,45 @@ function createTestTree(rootTestOptions, globalOptions) {
     snapshotManager: null,
     isFilteringByName,
     isFilteringByOnly,
+    async waitForGlobalSetup() {
+      if (globalSetupExecuted) {
+        globalSetupDeferred.resolve();
+        return;
+      }
+
+      const globalSetupFunctions =
```

Comment:
```suggestion
      const globalSetupFunctions =
```
This kind of formatting looks weird